packages/quantic/force-app/main/default/lwc/quanticDidYouMean/__tests__/quanticDidYouMean.test.js

```diff
@@ -1,10 +1,37 @@
 /* eslint-disable no-import-assign */
-// @ts-ignore
-import {createElement} from 'lwc';
 import QuanticDidYouMean from '../quanticDidYouMean';
 import * as mockHeadlessLoader from 'c/quanticHeadlessLoader';
+import {buildCreateTestComponent, cleanup, flushPromises} from 'c/testUtils';
 
 jest.mock('c/quanticHeadlessLoader');
+jest.mock(
+  '@salesforce/label/c.quantic_NoResultsFor',
+  () => ({default: 'No Results for {{0}}'}),
+  {
+    virtual: true,
+  }
+);
+jest.mock(
+  '@salesforce/label/c.quantic_QueryCorrectedTo',
+  () => ({default: 'Query was automatically corrected to {{0}}'}),
+  {
+    virtual: true,
+  }
+);
+jest.mock(
+  '@salesforce/label/c.quantic_ShowingResultsFor',
+  () => ({default: 'Showing results for {{0}}'}),
+  {
+    virtual: true,
```

Comment:
a tu besoin de tout ca ?

Reply:
Yea, otherwise we get no labels, otherwise the HTML will get rendered something like this:

`<div>@salesforce/label/c.quantic_ShowingResultsFor</div>`

But with the mock it gets rendered correctly as:

`<div>Showing results for MY_QUERY</div>` 

So we can properly test it.